Added simple example of Vaadin UI (#854)
vaadin/flow-and-components-documentation: documentation/spring/tutorial-spring-basic.asciidoc

[TIP]
Follow the instructions on https://docs.spring.io/spring-boot/docs/current/reference/html/howto-traditional-deployment.html[Spring Boot documentation], if you want to deploy your Spring Boot application as a traditional WAR file.

== Add Vaadin view to Spring Boot application

> With Vaadin views are defined as Java classes using `@Route` annotation. Component scan will detect the classes and publish them in a path derived from the class name or defined as parameter to the annotation. 

Reviewer: ```suggestion
With Vaadin views are defined as Java classes using `@Route` annotation. At the application start the classes are detected and published in a path derived from the class name or defined as a parameter to the annotation. 
```

> With Vaadin views are defined as Java classes using `@Route` annotation. Component scan will detect the classes and display them in given path. If you leave out the path parameter the view is served in the root context of the application, and you can specify multiple views by givin them different paths. 

Reviewer: ```suggestion
With Vaadin views are defined as Java classes using `@Route` annotation. Component scan will detect the classes and publish them in a path derived from the class name or defined as parameter to the annotation. 
```

*Example*: `MainView` class.

[source,java]
----
@Route
public class MainView extends VerticalLayout {

	public MainView() {
		add(new Text("Welcome to MainView."));
	}

}
----
[TIP]

> If no route path is provided, the path will be derived from the class name of the class. The derived name will be in lower case and trailing "View" will be removed. Also, `MainView` or `Main` names will be mapped to root (path will be `""`)

Reviewer: ```suggestion
If you don't provide a path parameter, the framework will derive the path from the class name of the class. The derived name will be in lower case, and possible trailing "View" will be removed. Also, `MainView` or `Main` names will be mapped to root (path will be `""`)
```
